apilayer/handler/UserHandler.go

```diff
@@ -216,6 +217,65 @@ func IsValidUserEmail(rw http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(rw).Encode(resp)
 }
 
+// VerifyEmailAddress ...
+// swagger:route GET /api/v1/verify Authentication VerifyEmailAddress
+//
+// # Used to verify if the user correctly verified the selected email address. If the token
+// is valid, then the user was successfully verified
+//
+// Responses:
+// 200: MessageResponse
+// 400: MessageResponse
+// 404: MessageResponse
+// 500: MessageResponse
+func VerifyEmailAddress(rw http.ResponseWriter, r *http.Request) {
+
```

Comment:
need to complete this

Reply:
Plus we might need to tweak this a little bit since we are going to be using jwt link and if verified the user is the correct user.